src/ls-alt.cpp

```diff
@@ -0,0 +1,22 @@
+#include "commands.h"""
+
+#include <string>
+#include <iostream>
+#include <filesystem>
+namespace fs = std::filesystem;
+
+
+namespace seashell {
+
+void ls(const std::string &)
+{
+	std::string path = fs::current_path();
+    
+	for (auto& entry : fs::directory_iterator(path))
+    
+	std::cout << entry.path() << std::endl;
+	
+}
+}
+
+

```

Comment:
This entire file is unneeded, as the #11 branch was merged this morning.

The idea is better, but it does not have a guarantee of the sort order of the files.